manifest.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "Are men talking too much?",
+    "description": "Check who's dominating the conversation",
+    "start_url": "./",
+    "background-color": "#FFFFFF",
```

Comment:
Yes, sorry for the typo.
I just found this validator: https://manifest-validator.appspot.com/